python-threatexchange/threatexchange/exchanges/clients/stopncii/api.py

```diff
@@ -62,12 +61,11 @@ class StopNCIICSPFeedback:
     tags: t.Set[str] = field(default_factory=set)  # Unstructured additional tags
     source: str = ""  # Name of the Content Service Provider (CSP)
 
-    @classmethod
-    def as_dict_for_post(cls) -> t.Dict[str, t.Any]:
+    def as_dict_for_post(self) -> t.Dict[str, t.Any]:
         """The json-friendly format to send in post requests"""
         return {
-            "tags": list(cls.tags),
-            "feedbackValue": str(cls.feedbackValue),
+            "tags": list(self.tags),
+            "feedbackValue": self.feedbackValue.value,
```

Comment:
`str(self.feedbackValue)` looks like `StopNCIICSPFeedbackValue.Blocked`, which should be `Blocked`, derived from `enum.value`